apps/trustlab/src/lib/data/common/index.js

```diff
@@ -29,14 +29,16 @@ function getFooter(settings) {
     secondaryLogo,
     secondaryNavigation,
     title,
+    quickLinksLabel,
```

Comment:
Do we need this? I thought `label`/`title` will be `primaryNavigation.title || secondaryNavigation.title`.